training-server.yml

```diff
@@ -1,93 +1,156 @@
+        state: present
+
+    - name: Install Make Movie script Prerequisite | MEncoder - Package
+      become: yes
+      yum:
+        name: mencoder
+        state: present
+
+    - name: OMERO.figure server-side prerequisites, script prerequisites + web server for decoupled OMERO.web
+      become: yes
+      yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        # For OMERO.figure
+        - python-reportlab
+        - python-markdown
+        # For the 'make movie' script
+        - mencoder
+        - scipy
 
-- hosts: training-server
   roles:
-  - role: omero-server
-    # Taking this from the openstack playbook os-omero.yml, to install the systemd config.
-    omero_systemd_setup: True
+
+    - role: openmicroscopy.postgresql
+      postgresql_databases:
+      - name: omero
+      postgresql_users:
+      - user: omero
+        password: omero
+        databases: [omero]
+      postgresql_version: "9.6"
+
+    - role: openmicroscopy.omero-server
+      omero_server_release: 5.4.1
+
+    - role: openmicroscopy.omero-web
+      omero_web_release: 5.4.1
+      # omero_web_config_set:
+
+  post_tasks:
+    - name: Omero.web plugins | plugin install via pip & pypi
+      become: yes
+      pip:
+        name:
+        - "omero-figure==3.1.2"
+        - "omero-fpbioimage==0.2.0"
+        - "omero-webtagging-autotag==3.0.2"
+        - "omero-webtagging-tagsearch==3.0.3"
+        - "omero-iviewer==0.4.1"
+        editable: False
+        state: present
+        # variable comes from role openmicroscopy.omero-web
+        virtualenv: "{{ omero_web_basedir }}/venv"
+        virtualenv_site_packages: yes
+      notify:
+        - restart omero-web
+
+    - name: Create a figure scripts directory
+      become: yes
+      file:
+        path: /opt/omero/server/OMERO.server/lib/scripts/omero/figure_scripts
+        state: directory
+        mode: 0755
+        recurse: yes
+        owner: "omero-server"
+        group: "omero-server"
+
+    - name: Download the Figure_To_Pdf.py script
+      become: yes
+      get_url:
+        url: https://raw.githubusercontent.com/ome/omero-figure/v3.1.2/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py
+        dest: /opt/omero/server/OMERO.server/lib/scripts/omero/figure_scripts/Figure_To_Pdf.py
+        mode: 0755
+        owner: "omero-server"
+        group: "omero-server"
+
+    - name: Create a cambridge_scripts directory
+      become: yes
+      file:
+        path: /opt/omero/server/OMERO.server/lib/scripts/omero/cambridge_scripts
+        state: directory
+        mode: 0755
+        recurse: yes
+        owner: "omero-server"
+        group: "omero-server"
+
+    - name: Download the Scipy_Gaussian_Filter.py script
+      become: yes
+      get_url:
+        url: https://raw.githubusercontent.com/jburel/omero-example-scripts/cambridge-training/cambridge-training-2017-12/Scipy_Gaussian_Filter.py
```

Comment:
I guess this will (eventually) end up in an org repo?

Reply:
it will 
unfortunately we did not have time to set repo etc.
too many moving parts